lib/spack/spack/test/cmd/uninstall.py

```diff
@@ -13,6 +15,9 @@
 uninstall = SpackCommand('uninstall')
 install = SpackCommand('install')
 
+pytestmark = pytest.mark.skipif(sys.platform == "win32",
+                                reason="does not run on windows")
+
```

Comment:
I think we may want to be able to test at least the most basic uninstall functionality on Windows, but to check: is it that `spack uninstall` works but that the tests seem to fail, or is it that neither seems to work?

Reply:
This is being handled as per Slack.